automation/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.4.0
\ No newline at end of file
+sbt.version = 1.4.6
```

Comment:
The other files seem to be newline-terminated. I'd do the same here for consistency (and to play nicely with command line commands)